macro/utils/src/lib.rs

```diff
@@ -0,0 +1,69 @@
+//! Module for various functions and structs to build macros in iroha.
```

Comment:
Might there be done other functions that we use across crates that could go in here?

Reply:
Probably, my initial idea was that i would place here what i need to reuse. 
And in the future when working with other macros common functionality would emerged it could be extracted here.